_dashboards/management/advanced-settings.md

```diff
@@ -6,18 +6,151 @@ nav_order: 40
+`fields:popularLimit` | Defines the number of N fields to be displayed. Default is `10`.  |
+`filterEditor:suggestValues` | Defines whether the filter editor suggests field values. Default is `Off`.  |
+`filters:pinnedByDefault`  | Defines whether filters are automatically pinned. Default is `Off.`  |
+`format:bytes:defaultPattern`  | Defines the default numeral format for the bytes format. Default is `0,0.[0]b`.  |
+`format:currency:defaultPattern` | Defines the default numeral format for the currency format. Default is `($0,0.[00])`.  |
+`format:defaultTypeMap` | Defines the default format name for each field type using a mapping. If the field type isn't specified, `_default_` is used.  |
+`format:number:defaultLocale`  | Defines the language locale for numerals. Default is `en`.  |
+`format:number:defaultPattern`  | Defines the default numeral format for the number format. Default is `0,0.[000]`.  |
+`format:percent:defaultPattern`  | Defines the default numeral format for the percent format. Default is `0,0.[000]%`.  |
+`histogram:barTarget`  |  Defines a specified number of bars for date histograms that use the `auto` interval. Default is `50`.  |
+`histogram:maxBars`  | Defines the maximum number of bars to show in date histograms. Default is `100`.  |
+`history:limit` | Defines how many of the most recent values to store in the history. Default is `10`.  |
+`indexPattern:placeholder`  | Defines the placeholder value to use when creating index patterns.  |
+`metaFields` | Enables fields that are not part of the `_source` field to be merged into the document. Default is `_source`, `_id`, `_type`, `_index`, and `_score`.  |
+`metrics:max_buckets` | Defines the maximum number of buckets that a single data source can return. Default is `2000`.  |
+`query:allowLeadingWildcards`  | Defines whether `*` is allowed as the first character in a query clause. Default is `On`.  | 
+`query:queryString:options`  |  Defines the options for the Lucene query string parser. Default is `true`.  |
+`reporting:useFOR`  | Enables or disables `ForeignObject` rendering for embedding of external content into reports. Default is `On`.  |
+`reporting:useOcr`  | Enables or disables optical character recognition (OCR) to be run on PRF reports. Default is `Off`.  |
+`savedObjects:listingLimit`  | Defines the number of objects to fetch when viewing a listing page. Default is `1000`.  |
+`savedObjects:perPage`  | Defines the number of objects to display on each page of the load dialog. Default is `20`.  |
+`search:queryLanguage`  | Defines the query language for OpenSearch Dashboards. Default is `DQL`.  |
+`shortDots:enable`  | Enables or disables the shortening of long fields. Default is `Off`.  |
+`sort:options`  | Defines the options for the sort parameter. Default is `boolean`.  |
+`state:storeInSessionStorage`  | Enables or disables the storing of URLs. Default is `Off`.  |
```

Comment:
The definitions here are not accurate: URLs are always stored. The default is off which means the URLs will not be stored in session storage, but will be stored in browsers.

Reply:
@abbyhu2000 Revised. Please review and make any necessary changes.